### ISSUE TITLE
#-943847739------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1565]
<strong>Source:</strong>`<a id="edit-button-434c061b20f74696a4619513b2811ca1" href="/layers/434c061b20f74696a4619513b2811ca1/edit" aria-describedby="calcite-tooltip-080de8b2-090d-9dd0-a624-d709ad1052f0">`

<strong>Element Location: </strong>#edit-button-434c061b20f74696a4619513b2811ca1

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-name?application=axeAPI
